test/rubygems/test_gem_specification.rb

```diff
@@ -2311,18 +2311,18 @@ def test_to_ruby_for_cache
 # stub: a 2 ruby lib
 
 Gem::Specification.new do |s|
-  s.name = "a"
+  s.name = "a".freeze
   s.version = "2"
```

Comment:
No particular reason. This PR is an experiment to see a) how @drbrain feels about it, and b) whether it has any *real* impact.